config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/scientists/:id", to: "scientists#show"
+  delete "/scientists/:id", to: "scientists#destroy"
+
+  get "/experiments", to: "experiments#index"
```

Comment:
Your GET routes are great, your DELETE route is not exactly RESTful. We are not deleting a scientist from the database, what we're doing is deleting a relationship between a scientist and an experiment. For this, this route should be a nested route, being sent to a nested controller.